docs/contrib/PlatformBringUp.md

```diff
@@ -0,0 +1,415 @@
+		- [Platform Directory](#platform-directory)
+		- [Design Directory](#design-directory)
+	- [Platform Configuration](#platform-configuration)
+	- [Design Configuration](#design-configuration)
+		- [config.mk](#configmk)
+		- [constraint.sdc](#constraintsdc)
+		- [Liberty, LEF, and GDS Files](#liberty-lef-and-gds-files)
+	- [Behavioral Models](#behavioral-models)
+		- [Gated Clock](#gated-clock)
+		- [Latch](#latch)
+	- [FastRoute Configuration](#fastroute-configuration)
+	- [Metal Tracks Configuration](#metal-tracks-configuration)
+	- [PDN Configuration](#pdn-configuration)
+	- [Tapcell Configuration](#tapcell-configuration)
+	- [setRC Configuration](#setrc-configuration)
+	- [KLayout](#klayout)
+		- [KLayout properties file](#klayout-properties-file)
+		- [KLayout tech file](#klayout-tech-file)
+- [Validating the New Platform](#validating-the-new-platform)
+- [Authors/Contributors](#authorscontributors)
+
+
+# Overview
+
+This document is a guide for foundry and third party IP providers to easily integrate and test a new technology in to the OpenROAD RTL to GDS flow. OpenROAD allows you to integrate any PDK (Process Design Kit) for any feature size and implement a fully open-sourced RTL-GDSII flow (synthesizable Verilog to merged GDSII). The OpenROAD flow has been validated for feature sizes down to 7nm and used to design and tapeout over 100 ASIC and SoCs to date.
+
+
+# Prerequisites
+
+To build and add a new platform for OpenROAD, key technology and library components must be provided based on the technology node. These are generally available as part of the standard design kit provided by a foundry or a third-party IP provider.  
+
+They include :
+
+* A standard cell library
+    * GDS files of all standard cells in the library (or a way to generate them from the layout files, e.g., Magic VLSI layout tool)
+* A technology LEF file of the PDK being used that includes all relevant information regarding metal layers, vias, and spacing requirements. 
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lef/NangateOpenCellLibrary.tech.lef` as an example tech LEF file.
+* A macro LEF file of the standard cell kit that includes MACRO definitions of every cell, pin designations (input/output/inout).
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lef/NangateOpenCellLibrary.macro.lef `as an example macro LEF file.
+* A Liberty file of the standard cell library with PVT characterization, input and output characteristics, timing and power definitions for each cell.
+    * See `OpenROAD-flow-scripts/flow/platforms/nangate45/lib/NangateOpenCellLibrary_typical.lib `as an example liberty file.
+* For KLayout: A mapping from LEF/DEF to GDS layers:datatypes
+
+Adding a new platform additionally requires the following: 
+
+* A validated installation of the OpenROAD flow scripts is available. See instructions [here](https://openroad.readthedocs.io/en/latest/user/GettingStarted.html#building-and-installing-the-software).:
```

Comment:
Can this be a link to the GettingStarted,md?

Reply:
This is a link to GettingStarted.md - it's just the relevant section in the doc.

Reply:
I think this needs to be a relative link. Otherwise once it will always point to `latest` which could be a different version than this doc.

Reply:
You can try some of these approaches @rjridle [here](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#).

Reply:
Changed to relative link.